### PR TITLE
refactor: add team distribution student

### PR DESCRIPTION
**Pull request by valerydluski**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [x] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

_Describe the source of requirement, like related issue link._

#### 💡 Background and solution

_Describe the big picture of your changes here_

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: nestjs/src/courses/team-distribution/dto/team-distribution-student.dto.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { PaginationMetaDto } from 'src/core/paginate/dto/Paginate.dto';
 export class TeamDistributionStudentDto {
   constructor(student: Student) {
     this.id = student.id;
-    this.fullName = `${student.user.firstName} ${student.user.lastName}`;
+    this.fullName = `${student.user.firstName ?? ''} ${student.user.lastName ?? ''}`;
```
Comment (nataliagulko) [inline nestjs/src/courses/team-distribution/dto/team-distribution-student.dto.ts:9]: PersonDto has `getName` method. You can try to use it instead

File: nestjs/src/courses/team-distribution/dto/join-team.dto.ts [hunk 2]
```diff
@@ -12,8 +12,12 @@ export class JoinTeamDto {
 export class JoinTeamDtoRes {
```
Comment (nataliagulko) [inline nestjs/src/courses/team-distribution/dto/join-team.dto.ts:12]: Res - is "response", "result" or something else?

**Review by nataliagulko — APPROVED** [review]

File: nestjs/src/courses/team-distribution/dto/team-distribution-student.dto.ts (hunk 1 above)
Comment (valerydluski) [reply nestjs/src/courses/team-distribution/dto/team-distribution-student.dto.ts]: fixed

File: nestjs/src/courses/team-distribution/dto/join-team.dto.ts (hunk 2 above)
Comment (valerydluski) [reply nestjs/src/courses/team-distribution/dto/join-team.dto.ts]: fixed

**Review by apalchys — APPROVED** [review]

**Pull request merged**